book/website/reproducible-research/compendia.md

```diff
@@ -1,6 +1,6 @@
 # Research Compendia
 
-## Prerequisites / recommended skill level
+## Prerequisites / Recommended Skill Level
```

Comment:
This should be updated to only 'Prerequisite' as per our [new template](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/templates/chapter-template/chapter-landing-page.md).